.github/workflows/learn-github-actions.yml

```diff
@@ -0,0 +1,13 @@
+name: learn-github-actions
+run-name: ${{ github.actor }} is learning GitHub Actions
+on: [push]
+jobs:
+  check-bats-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '14'
```

Comment:
- to install the specified version of the Node.js.
- This puts both the `node` and `npm` commands in your` PATH`.